### PR TITLE
[DOCS] Alerts table updates (RAC)

### PR DESCRIPTION
**Pull request by joepeeples**
Addresses #991.

The Alerts page shows up in screenshots on several docs pages, so they all needed updating. Preview pages: 
- [Manage detection alerts](https://security-docs_1028.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html) — Main docs page for Alerts page
- [Elastic Security UI | Alerts page](https://security-docs_1028.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html#detection-engine-ui)
- [Detections and alerts](https://security-docs_1028.docs-preview.app.elstc.co/guide/en/security/master/detection-engine-overview.html)
- [Anomaly Detection with Machine Learning | Manage machine learning jobs](https://security-docs_1028.docs-preview.app.elstc.co/guide/en/security/master/machine-learning.html#manage-jobs)
- [Visual event analyzer](https://security-docs_1028.docs-preview.app.elstc.co/guide/en/security/master/visual-event-analyzer.html) — Not alerts per se, but the Events tab also changed to use the new Elastic UI data grid

### CODE REVIEW

**Comment by karanbirsingh-qasource** [comment]
Hi @joepeeples 

we have validated the above shared preview doc links and found some of the changes to be done .

Please find the details for the same 
https://github.com/elastic/security-docs/issues/1049
https://github.com/elastic/security-docs/issues/1050
https://github.com/elastic/security-docs/issues/1051

thanks !!!

c.c. @jmikell821

**Review by nastasha-solomon — APPROVED** [review]
Don't see any glaring issues so LGTM!

File: docs/detections/alerts-ui-manage.asciidoc [hunk 1]
```diff
@@ -22,9 +22,10 @@ NOTE: KQL autocomplete for `.siem-signals-*` indices is available on the
...
-* Use the *Stack by* drop-down in the Trend histogram to select specific parameters to visualize the individual counts. For example, if you select `signal.rule.name`, the histogram displays the total counts by alert name.
-* Filter alert results to include building block alerts or to only show alerts from indicator match rules, select the *Additional filters* drop-down. By default, building block alerts are excluded from the Alerts table; therefore, including them expands the number of alerts.
+* View detection alerts generated by a specific rule. To do this, go to *Detect* -> *Rules*, and then select a rule name in the All rules table. The *Rules detail page* displays a comprehensive view of the rule's details, and alert details are displayed in the Alerts table beneath the Trend histogram. Alternatively, you can go to *Detect* -> *Alerts* -> *Manage rules*, and then click on a rule name in the All rules table to view the alerts associated with the rule.
+* Visualize the alerts by specific parameters by using the *Stack by* drop-down in the Trend histogram. For example, if you select `signal.rule.name`, the histogram displays the total counts by rule name.
```
Comment (jmikell821) [inline docs/detections/alerts-ui-manage.asciidoc:26]:
```suggestion
* Visualize alerts by specific parameters by using the *Stack by* drop-down in the Trend histogram. For example, if you select `signal.rule.name`, the histogram displays the total alert counts by rule name.
```

File: docs/detections/alerts-ui-manage.asciidoc [hunk 2]
```diff
@@ -113,47 +114,51 @@ NOTE: The event enrichment query uses the indices specified in the `securitySolu
...
 
 [float]
 [[detection-alert-status]]
 === Change an alert's status
 
 You can set an alert's status to indicate whether it needs to be investigated
-(*Open*), is under active investigation (*In progress*), or resolved
-(*Closed*). By default, the Alerts table displays open alerts. To filter alerts that are *In progress* or *Closed*, select the appropriate status in the upper-right corner of the Alerts table.
+(*Open*), is under active investigation (*Acknowledged*), or has been resolved
+(*Closed*). By default, the Alerts page displays open alerts. To filter alerts that are *Acknowledged* or *Closed*, select the appropriate status in the upper-left corner of the Alerts page. This filters all of the results displayed on the page (the Trend histogram, and the Counts table, and the Alerts table).
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/detections/alerts-ui-manage.asciidoc:132]: Janeen confirmed with team that the table is called "Counts" for now.

**Pull request merged**